docs/README.md

```diff
@@ -114,35 +114,35 @@ If you're beginner, take a look at the step by step guide on how to contribute t
       <tr>
         <td align="center" valign="top" width="14.285714285714285714285714285714%">
           <a href="https://twitter.com/krakenfx" target="_blank">
-            <img  src="https://raw.githubusercontent.com/btcpayserver/btcpayserver-doc/master/docs/img/kraken-logo-stacked-purple-RGB-2019.svg?sanitize=true" alt="Kraken" height=100>
+            <img  src="https://raw.githubusercontent.com/btcpayserver/btcpayserver-doc/master/docs/img/kraken-logo-stacked-purple-RGB-2019.svg?sanitize=true" alt="Kraken" loading="lazy" height=100>
```

Comment:
The `loading="lazy"` attribute should be fine, as it is a native feature – in case the browser doesn't support it it loads the image directly. AFAIK the lazy loading on the blog was implemented via custom javascript, which doesn't work. btw: Who's on it?

Reply:
Who's on what ?

Reply:
Fixing the supporter images on the blog

Reply:
I have no idea, and I don't expect to be able to help in that regard.

Reply:
kukks fixed it :)